eng/packaging.props

```diff
@@ -15,6 +15,10 @@
     <SkipGenerationCheck>true</SkipGenerationCheck>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <NuSpecOutputPath Condition="'$(NuSpecOutputPath)' == ''">$([MSBuild]::NormalizeDirectory('$(ArtifactsPackagesDir)', 'specs'))</NuSpecOutputPath>
```

Comment:
Are nuspecs generated by default by NuGet when invoking their Pack task?

Reply:
Yes, by default its in the obj directory.